zipper/protocols/prometheus/helpers/helpers.go

```diff
@@ -139,29 +139,42 @@ func AdjustStep(start, stop, maxPointsPerQuery, minStep int64, forceMinStepInter
 // PromethizeTagValue - accept 'Tag=value' or 'Tag=~value' string and return sanitized version of it
 func PromethizeTagValue(tagValue string) (string, types.Tag) {
 	// Handle = and =~
-	t := types.Tag{}
-	idx := strings.Index(tagValue, "=")
-	if idx != -1 {
-		if tagValue[idx+1] == '~' {
+	var (
+		t       types.Tag
+		tagName string
+		idx     = strings.Index(tagValue, "=")
```

Comment:
I'm not a big fan of using val() block to initialize valuies, but I don't think it is a major problem of the PR. More of a FIY for next time.